Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift

```diff
@@ -36,6 +38,20 @@ struct DashboardView: View {
                         } label: {
                             Image(systemName: "trash")
                         }
+                        
+                        Button {
+                            selectedMaintenanceEvent = event
+                            isShowingEditView = true
+                        } label: {
+                            VStack {
+                                Text("Edit")
+                                Image(systemName: "pencil")
+                            }
```

Comment:
You can achieve the same thing with a `Label`, and it's better accessibility wise

`Label("Edit", systemImage: "pencil")`